### PR TITLE
feat: add GetEventInputName func for context interface

### PR DESCRIPTION
**Pull request by zclonely**

### CODE REVIEW

**Comment by benjaminhuo** [comment]
@zclonely You need to sign your commit:
```shell
git commit -s --amend
git push -f
```

**Review by benjaminhuo — APPROVED** [review]

File: context/context.go [hunk 1]
```diff
@@ -194,6 +194,9 @@ type Context interface {
 
 	// ContextOptions returns the context's options.
 	ContextOptions() ContextOption
+
+	// GetInputNameForEvent return the inputName of the event
+	GetInputNameForEvent() string
```
Comment (tpiperatgod) [inline context/context.go:199]: Could it be "GetInputName" or "GetEventInputName"?

Comment (benjaminhuo) [reply context/context.go:199]: GetInputName is better, @zclonely would you change this accordingly?

Comment (zclonely) [reply context/context.go:199]: No problem. I will change it immediately!  (`GetInputNameForEvent` to `GetInputName`)
BTW, Is there any relevant documentation need update？

Comment (benjaminhuo) [reply context/context.go:199]: No docs need to be updated so far

File: context/context.go [hunk 2]
```diff
@@ -521,10 +524,12 @@ func (ctx *FunctionContext) SetSyncRequest(w http.ResponseWriter, r *http.Reques
 }
 
 func (ctx *FunctionContext) SetEvent(inputName string, event interface{}) {
+	// klog.Errorf(inputName)
```
Comment (benjaminhuo) [inline context/context.go:527]: `// klog.Errorf(inputName)` should be removed?

File: context/context.go [hunk 3]
```diff
@@ -521,10 +524,12 @@ func (ctx *FunctionContext) SetSyncRequest(w http.ResponseWriter, r *http.Reques
...
 	switch t := event.(type) {
 	case *common.BindingEvent:
 		be := event.(*common.BindingEvent)
 		ie := convertEvent(ctx, inputName, be.Data)
+
```
Comment (benjaminhuo) [inline context/context.go:532]: This blank line should also be removed?
image

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**